docs/user/integration/dynatrace/README.md

```diff
@@ -93,10 +93,10 @@ Follow the instructions in [Dynatrace: Generate an access token](https://docs.dy
 
 ### Create Secret
 
-To create a new Secret containing your access token, replace the `{API_TOKEN}` placeholder with the token you created and run the following command:
+To create a new Secret containing your access token, replace the `{API_TOKEN}` placeholder with the token you created, replace the `{API_URL}` placeholder with the Dynatrace SaaS endpoint and run the following command:
 
 ```bash
-kubectl -n $DYNATRACE_NS create secret generic dynatrace-token --from-literal="apiToken=Api-Token {API_TOKEN}"
+kubectl -n $DYNATRACE_NS create secret generic dynakube --from-literal="apiToken=<API_TOKEN>" --from-literal="apiurl=<API_URL>"
```

Comment:
What's with the dataIngestToken? That needs to be used in our pipelines, isn't it?